sdk/monitor/monitor-query/test/public/logsQueryClient.spec.ts

```diff
@@ -38,13 +41,14 @@ describe("LogsQueryClient live tests", function() {
     try {
       // TODO: there is an error details in the query, but when I run an invalid query it
       // throws (and ErrorDetails are just present in the exception.)
-      await createClient().query(monitorWorkspaceId, kustoQuery, {
+
+      await logsClient.query(monitorWorkspaceId, kustoQuery, {
         duration: Durations.OneDay
       });
       assert.fail("Should have thrown an exception");
     } catch (err) {
       // eslint-disable-next-line @typescript-eslint/no-unused-vars -- eslint doesn't recognize that the extracted variables are prefixed with '_' and are purposefully unused.
-      const { request: _request, response: _response, ...stringizableError } = err;
+      const { request: _request, response: _response, ...stringizableError }: any = err;
```

Comment:
you don't have the type?

Reply:
I am not typecasting it to a type so that I can check if it has the innerError fields in the later steps.